### PR TITLE
give more explicit pointer to the contribute guide

### PR DESCRIPTION
**Pull request by maxandersen**
I would prefer the relevant `mvn` commands to use/run was documented in the repo rather than "off-site" but with this there is at least a chance to discover the documentation.

### CODE REVIEW

**Comment by maxandersen** [comment]
relates to https://github.com/quarkusio/quarkus/issues/29907

**Comment by ebullient** [comment]
Just add an additional sentence or direct link to the adoc file that is in the same directory... 

The pretty version is rendered on the web.. but the adoc is perfectly readable and is right there...

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## Quarkus Documentation
+
+This is the main folder for Qaurkus documentation. 
+
+Details on how to contribute documentation can be found at https://quarkus.io/guides/doc-contribute-docs-howto.
```
Comment (ebullient) [inline docs/README.md:5]:
```suggestion
Details for how to contribute to Quarkus docs are found in the following locations: 

- [Quarkus documentation concepts](doc-concepts.adoc) (Diataxis framework): 
  - https://quarkus.io/guides/doc-concepts
- [How to contribute](doc-contribute-docs-howto.adoc): 
  - https://quarkus.io/guides/doc-contribute-docs-howto
- [About Quarkus documentation](doc-reference.adoc):
	- https://quarkus.io/guides/doc-reference
- [Tutorial example: Create a tutorial](doc-create-tutorial.adoc): 
  - https://quarkus.io/guides/doc-create-tutorial
```

**Comment by maxandersen** [comment]
> Just add an additional sentence or direct link to the adoc file that is in the same directory...
> 
> The pretty version is rendered on the web.. but the adoc is perfectly readable and is right there...

which adoc file? the docs folder has no such file.

**Comment by ebullient** [comment]
See the suggested edits. All of those files exist in docs/src/asciidoc.

**Comment by maxandersen** [comment]
> See the suggested edits. All of those files exist in docs/src/asciidoc.

ok, so not the same folder. `docs` != `docs/src/asciidoc`  :)

**Comment by maxandersen** [comment]
push update that points out the nested readme.adoc to avoid duplication and broken links.

**Review by ebullient — APPROVED** [review]

**Comment by ebullient** [comment]
squash please.

File: docs/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+## Quarkus Documentation
+
+This is the main folder for Qaurkus documentation. 
+
+This folder is synchronized to website and the [Primary Documentatin README.adoc](src/main/asciidoc/README.adoc) are inside the `src/main/asciidoc` folder.
```
Comment (inoxx03) [inline docs/README.md:5]:
```suggestion
This folder is synchronized to website and the [Primary Documentation README.adoc](src/main/asciidoc/README.adoc) are inside the `src/main/asciidoc` folder.
```

File: docs/README.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+## Quarkus Documentation
+
+This is the main folder for Qaurkus documentation.
```
Comment (inoxx03) [inline docs/README.md:3]:
```suggestion
This is the main folder for Quarkus documentation. 
```

**Comment by inoxx03** [comment]
@maxandersen thank you for the update. I spotted some typos and dropped in edit suggestions to fix. Other than that, LGTM.

File: docs/README.md (hunk 2 above)
Comment (inoxx03) [reply docs/README.md:5]: Suggesting and alternative phrasing. I leave it up to you to accept or decline this change.
```suggestion
The contents of this folder are used to publish documentation to the quarkus.io/docs website.
The [main documentation README.adoc](src/main/asciidoc/README.adoc) is in the `src/main/asciidoc` folder that also contains the `.adoc` source files for the published documentation.
```

**Review by inoxx03 — APPROVED** [review]
Added some phrasing suggestions, too. Feel free to accept/decline them as you see fit, but please fix the typos:)

**Review by gsmet — APPROVED** [review]
Squashed and tuned some additional things.

**Pull request merged**